resource-server/okta-aspnet-webapi-example/Web.config

```diff
@@ -6,231 +6,231 @@
 <configuration>
   <appSettings>
     <!-- 1. Replace these values with your Okta configuration -->
-    <add key="okta:ClientId" value="{clientId}" />
-    <add key="okta:OktaDomain" value="https://{yourOktaDomain}.com/" />
+    <add key="okta:OktaDomain" value="https://{yourOktaDomain}" />
   </appSettings>
+  <!--
+    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.
+
+    The following attributes can be set on the <httpRuntime> tag.
+      <system.Web>
+        <httpRuntime targetFramework="4.7" />
+      </system.Web>
+  -->
   <system.web>
-    <compilation debug="true" targetFramework="4.6.1" />
+    <compilation debug="true" targetFramework="4.7" />
```

Comment:
Are we sure we want to update this?  Any possible issues we should be aware of?

Reply:
It shouldn't cause any issues. Actually, I'm going to create an issue to update the readme with the versions we support and use the newest supported version in our samples.